packages/expo-localization/README.md

```diff
@@ -96,55 +109,76 @@ export default class LitView extends React.Component {
 
 ## API
 
+This API is mostly synchronous and driven by constants. On iOS the constants will always be correct, on Android you should check if the locale has updated using `AppState` and `Localization.getLocalizationAsync()`. Initally the constants will be correct on both platforms, but on Android a user can change the language and return, more on this later.
+
 ### Constants
 
-This API is mostly synchronous and driven by constants. On iOS the constants will always be correct, on Android you should check if the locale has updated using `AppState` and `Expo.Localization.getLocalizationAsync()`. Initally the constants will be correct on both platforms, but on Android a user can change the language and return, more on this later.
+| Name             | Type       | Description                                                                                                                                  | iOS | Android | Web |
+| ---------------- | ---------- | -------------------------------------------------------------------------------------------------------------------------------------------- | --- | ------- | --- |
+| locale           | `string`   | Native device language, returned in standard format. Ex: `en`, `en-US`, `es-US`.                                                             | ✅  | ✅      | ✅  |
+| locales          | `string[]` | List of all the native languages provided by the user settings. These are sorted in the order the user has defined in their native settings. | ✅  | ✅      | ✅  |
+| country          | `string?`  | Country code for your device.                                                                                                                | ✅  | ✅      | ✅  |
+| timezone         | `string`   | The current time zone in display format. ex: `America/Los_Angeles`. Read more about web usage below.                                         | ✅  | ✅      | ✅  |
+| isRTL            | `boolean`  | This will return `true` if the current language is Right-to-Left.                                                                            | ✅  | ✅      | ✅  |
+| isoCurrencyCodes | `string[]` | A list of all the supported ISO codes.                                                                                                       | ✅  | ✅      | ❌  |
 
-#### `Localization.locale: string`
+**Web Timezone**
 
-Native device language, returned in standard format. Ex: `en`, `en-US`, `es-US`.
+On web we will attempt to get the timezone with the native `Intl.DateTimeFormat().resolvedOptions().timeZone` but if cannot be found, the default value `Etc/UTC` will be returned. For more accurate results you could use the library `moment/timezone`. Because [`moment.js` is so large](https://github.com/moment/moment/issues/3376) we don't include it.
```

Comment:
```suggestion
On web we will attempt to get the timezone with the standard `Intl` API but if the time zone cannot be found, the default value `Etc/UTC` will be returned. For more accurate results you could use the library `moment/timezone`. Because [`moment.js` is so large](https://github.com/moment/moment/issues/3376) we don't include it.
```